### ISSUE TITLE
little bug for log out account

### ISSUE THREAD

**Issue by XiaZhenchao** [CUNY-Computer-Vision-Lab/rd-ar-annotate#22]
Users can't log out accounts.  In the second view, there has a Login button in the navigation bar, when I click it, it will jump to login view but soon it will jump back to the second view. The user can't log out the account through click Login button in CreateAnnotationViewController this view.

**Comment by tyfranklin** [comment]
Good catch! I think I only made the login view. The only real logout is a hard reset. This can be the ticket to make the logout view.
